### ISSUE TITLE
ExternalMethod ZMI problem

### ISSUE THREAD

**Issue by georgpfolz** [zopefoundation/Zope#635]
Zope 4.0: ExternalMethods work (can be created, imported and run) but cannot be accessed in the ZMI:

```
Traceback (innermost last):

Module ZServer.ZPublisher.Publish, line 144, in publish
Module ZPublisher.mapply, line 85, in mapply
Module ZServer.ZPublisher.Publish, line 44, in call_object
Module Shared.DC.Scripts.Bindings, line 335, in __call__
Module Shared.DC.Scripts.Bindings, line 372, in _bindAndExec
Module App.special_dtml, line 214, in _exec
Module DocumentTemplate._DocumentTemplate, line 144, in render_blocks
Module DocumentTemplate._DocumentTemplate, line 166, in render_blocks_
Module DocumentTemplate._DocumentTemplate, line 375, in __getitem__
Module DocumentTemplate._DocumentTemplate, line 393, in getitem
Module Shared.DC.Scripts.Bindings, line 349, in __render_with_namespace__
Module Shared.DC.Scripts.Bindings, line 372, in _bindAndExec
Module App.special_dtml, line 214, in _exec
Module DocumentTemplate._DocumentTemplate, line 144, in render_blocks
Module DocumentTemplate._DocumentTemplate, line 236, in render_blocks_
Module DocumentTemplate._DocumentTemplate, line 236, in render_blocks_
Module DocumentTemplate._DocumentTemplate, line 222, in render_blocks_
Module DocumentTemplate.DT_Util, line 228, in eval
__traceback_info__: getBookmarkableURLs
Module <string>, line 1, in <module>
NameError: name 'getBookmarkableURLs' is not defined
```

Worked with Zope 4.0b9 and b10. Same Version of Products.ExternalMethod: 4.1 (same problem with 4.2)

**Comment by d-maurer** [comment]
`getBookmarkableURLs` is a new feature.

Unlike other objects, `ExternalMethod` inherits from `Acquisition.Explicit` and not (`Acquisition.Implicit`). This means, they cannot implicitly access acquired objects. Provision to access `getBookmarkableURLs` was still missing.

I have just pushed a fix to branch `acquire_getBookmarkableURLS` of repository https://github.com/zopefoundation/Products.ExternalMethod. I am too lazy to set up the environment to check whether this fix resolves the issue. Maybe, you can make this check.

**Comment by dataflake** [comment]
This is already fixed in e7192fbd55aa9120b0c5b425e71034fa53bf01c0 and will be part of Zope 4.0.1

**Comment by dataflake** [comment]
I have also just released Products.ExternalMethod version 4.3 with Dieter's patch, which should solve it immediately.
